docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,187 @@
+- look at each SHW equipment in the building: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - CONTINUE TO RULE LOGIC
+- there are no cases where a Service Water Heating Equipment is not applicable to the rule
+- if there are no SWH equipment in the RMD, then this rule is not applicable: `if(len(B_RMD.service_water_heating_equipment) == 0): NOT APPLICABLE`
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = ""`
+  - we need to find the following information about the SWH:
+    - swh_type - one of the ServiceWaterHeaterTankOptions
+    - input_capacity (btu/hr)
+    - storage_volume (gallons)
+    - draw_pattern (VERY_SMALL, LOW, MEDIUM, or HIGH)
+    - capacity_per_volume (btu/hr/gallon)
+  - get the service water heating system: `swh_eq = get_object_by_id(service_water_heating_equipment_id,RMD)`
+  - get the type of tank: `tank_type = swh_eq.tank.type`
+  - the swh_type will be one of the ServiceWaterHeaterTankOptions, which are more detailed than the four types we need.  Determine our type based on the detailed type.
+  - the default value of swh_type is OTHER.  If the swh_type can be identified, the value will be changed in the logic below: `swh_type = "OTHER"`
+  - if the tank_type is CONSUMER_INSTANTANEOUS, COMMERCIAL_INSTANTANEOUS, or RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS, set the swh_type to INSTANTANEOUS and go to rule assertion: `if tank_type in ["CONSUMER_INSTANTANEOUS", "COMMERCIAL_INSTANTANEOUS", "RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS"]: swh_type = "INSTANTANEOUS"`; GO TO RULE_ASSERTION
+
+  - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+  - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the swh_type to STORAGE for the fuel type: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]:`
+    - set the swh_type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY": swh_type = "ELECTRIC_RESISTANCE_STORAGE`
+    - `elif fuel_type == "NATURAL_GAS": swh_type = "GAS_STORAGE`
+    - if the fuel type is propane, this is treated as natural gas in Table 7.8: `elif fuel_type == "PROPANE": swh_type = "GAS_STORAGE"`
+    - if the fuel type is not Electricity, Natural Gas, or Propane, this is not a valid baseline SWH system.  Create a rule note and go directly to the rule assertion: `else: rule_note += "Fuel type: " + fuel_type + " is not a valid fuel type for a service water heating baseline system.  According to ASHRAE 90.1 Table G3.1.1-2 service water heating equipment shall be either electric resistance or natural gas.  According to ASHRAE 90.1 Table G3.1 #11 h, in cases where natural gas is specified as the baseline system, but there is no natural gas available on site, a propane system may be modeled.  "; rule_status = FAIL; GO TO RULE ASSERTION `
```

Comment:
Is this FAIL duplicating a FAIL received in rule 11-7 due to the fuel type not matching? if so, suggest UNDETERMINED